.github/workflows/release_pipeline.yml

```
@@ -0,0 +1,320 @@
+            PLUGINS_STATUS="❌ Failed"
+          fi
+          
+          if [ "${{ needs.bifrost-http-release.result }}" = "success" ]; then
+            BIFROST_STATUS="✅ Released v${{ needs.detect-changes.outputs.transport-version }}"
+          elif [ "${{ needs.bifrost-http-release.result }}" = "failure" ]; then
+            BIFROST_STATUS="❌ Failed"
+          fi
+          
+          TITLE="🚀 **Release Pipeline Complete**"
+          MESSAGE="$TITLE\n**Core**: $CORE_STATUS\n**Framework**: $FRAMEWORK_STATUS\n**Plugins**: $PLUGINS_STATUS\n**Bifrost HTTP**: $BIFROST_STATUS\n**Branch**: \`main\`\n**Commit**: \`${{ github.sha }}\`\n**Author**: ${{ github.actor }}\n**[View Workflow Run](${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }})**"
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)